### ISSUE TITLE
Unable to override aura type in yamda dimensions (ftb omnia 1.15.2)

### ISSUE THREAD

**Issue by mmis1000** [Ellpeck/NaturesAura#116]
## Step to reproduce

1. download and start the `ftb omnia 1.15.2` server once to generate necessary files.
2. edit the naturesaura's config and change this line
  ```auraTypeOverrides = ["yamda:yamda_dim->naturesaura:overworld", "yamda:yamda_nether_dim->naturesaura:nether"]```
3. restart the server
4. use bottle in the yamda dimensions

## Expect

Get the correspond aura bottles because we already override them.

## Actually

Get `bottle of substance`

**Comment by Ellpeck** [comment]
Are you sure that that's the name of the dimension's type?

**Comment by mmis1000** [comment]
I did a `forge dimesions` to list all dimensions

```
[12May2020 17:33:58.684] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Currently registered dimensions by type:
[12May2020 17:33:58.684] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Vanilla: minecraft:overworld, minecraft:the_end, minecraft:the_nether
[12May2020 17:33:58.685] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: goodnightsleep:good_dream: goodnightsleep:good_dream
[12May2020 17:33:58.685] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: goodnightsleep:nightmare: goodnightsleep:nightmare
[12May2020 17:33:58.685] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: javd:void: javd:void
[12May2020 17:33:58.685] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: moolands:moolands: moolands:moolands
[12May2020 17:33:58.685] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: the_bumblezone:ultraamplified: the_bumblezone:bumblezone
[12May2020 17:33:58.685] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: yamda:yamda_dim: yamda:yamda_dim
[12May2020 17:33:58.685] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: yamda:yamda_nether_dim: yamda:yamda_nether_dim
```

This is what I get

**Comment by Ellpeck** [comment]
Please attach a log file, there should be some info in it about a failed config parse.

**Comment by mmis1000** [comment]
Uploaded (the file is a bit giant)
https://gist.github.com/mmis1000/23aa3635f9ff3e2ace7c82e38d873c9a

**Comment by Ellpeck** [comment]
Hmm. Can you try changing your `"double-quoted strings"` to `'single-quoted strings'` in the config value's array? I'm not sure if that'll make a difference, but it's worth a try.

**Comment by mmis1000** [comment]
Still not work, doesn't seems matter

**Comment by Ellpeck** [comment]
Could you try with this version and then send the log?
https://ci.ellpeck.de/job/NaturesAura/job/master/90/artifact/build/libs/NaturesAura-26.0.90.jar

**Comment by mmis1000** [comment]
Replaced the 23.0  jar with 26.0.90 jar you give, doesn't seems makes a different
https://paste.ee/p/ouGCC

**Comment by mmis1000** [comment]
Confirmed to be fixed in NaturesAura-27.0
